src/specs/components/news-ticker/demo/data.js

```diff
@@ -12,34 +12,53 @@ module.exports = {
   sr_pause: 'Pause news ticker',
   items: [
     {
-      content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+      content:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et finibus mi. Sed hendrerit velit id ex ornare interdum. Duis suscipit tortor vitae mi feugiat suscipit. Morbi consequat tortor magna, ac malesuada nunc placerat non. Vivamus pulvinar augue tristique lectus lobortis iaculis.',
       link: exampleLink,
+      icon: {
+        name: 'euro',
```

Comment:
maybe we should have different icons on some slides, to showcase this possibility

Reply:
Done it, although for some reasons i was explicitely requested to use the euro icon..